solvers/fast_osqp_solver.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
```

Comment:
bit of a nitpick but it might be worth adding a comment directing people to the drake OsqpSolver class for where this came from